specs/merge/beacon-chain.md

```diff
@@ -231,6 +236,7 @@ def process_execution_payload(state: BeaconState,
         timestamp=execution_payload.timestamp,
         receipt_root=execution_payload.receipt_root,
         logs_bloom=execution_payload.logs_bloom,
+        randao=execution_payload.randao,
```

Comment:
We should note somewhere here or in our own notes that this should probably be accompanied with an EIP. There are probably a handful of things that will need to make it into an EIP.